### PR TITLE
Tank heating + WDR Firmware Edit

### PR DESCRIPTION
**Pull request by JasonBrave**

### CODE REVIEW

File: src/clientside/config.cpp [hunk 1]
```diff
@@ -8,10 +8,15 @@ ActuatorMessage build_command_message() {
   return ActuatorMessage{
       .ov101 = digitalRead(pinout::MISSILE_SWITCH_1),
       .ov102 = digitalRead(pinout::MISSILE_SWITCH_2),
-      .nv101 = digitalRead(pinout::MISSILE_SWITCH_3),
-      .ov201 = digitalRead(pinout::MISSILE_SWITCH_4),
-      .ov202 = digitalRead(pinout::MISSILE_SWITCH_5),
-      .injector_valve = digitalRead(pinout::MISSILE_SWITCH_INJECTOR),
+      .nv201 = digitalRead(pinout::MISSILE_SWITCH_3),
+      .nv202 = digitalRead(pinout::MISSILE_SWITCH_4),
+	  .vent_valve = digitalRead(pinout::MISSILE_SWITCH_5),
```
Comment (BluCodeGH) [inline src/clientside/config.cpp:13]: Pls fix indentation, here and in other files.

File: src/clientside/config.cpp [hunk 2]
```diff
@@ -8,10 +8,15 @@ ActuatorMessage build_command_message() {
...
+	  .injector_valve = digitalRead(pinout::MISSILE_SWITCH_INJECTOR),
+	  .fill_dump_valve = digitalRead(pinout::MISSILE_SWITCH_6),
+      .fill_disconnect = digitalRead(pinout::MISSILE_SWITCH_7),
+	  .rocket_power = digitalRead(pinout::MISSILE_SWITCH_IGNITION_SEC), // Out of switch, should use omnibus dashboard
```
Comment (BluCodeGH) [inline src/clientside/config.cpp:17]: If we're going to control rocket power via dashboard (seems reasonable) then remove this line?

File: src/common/config.hpp [hunk 3]
```diff
@@ -22,10 +22,15 @@
 struct ActuatorMessage {
   bool ov101; // Ox Fill
   bool ov102; // Ox Drain
-  bool nv101; // N2 Purge
-  bool ov201; // Vent Isolation
-  bool ov202; // Vent Solenoid
-  bool injector_valve;
+  bool nv201; // N2 Purge
+  bool nv202; // N2 Drain (Solenoid)
```
Comment (BluCodeGH) [inline src/common/config.hpp:26]: I believe this is a motor valve and it's also QD disconnect, not just drain. Double check with Matthew tho.

File: src/towerside/actuators.hpp [hunk 4]
```diff
@@ -90,6 +90,112 @@ class Ignition: public I2C {
...
+public:
+
+  Heater(uint8_t slave_address):
+	  slave_address{slave_address},power_set{0},reg_set{0} {}
+
+  void set(bool value) {
+	power_set = value;
+    Wire.beginTransmission(slave_address);
+    // Relay boards have two relays. One turns on power, and the other selects which direction to apply power to.
+    // LSB is power, next bit is select.
```
Comment (BluCodeGH) [inline src/towerside/actuators.hpp:107]: These comments seem wrong, fix pls. Why are we writing the last used reg_set in this function anyways, that seems like its only relevant when reading?

**Review by BluCodeGH — CHANGES_REQUESTED** [review]

File: src/towerside/actuators.hpp (hunk 4 above)
Comment (JasonBrave) [reply src/towerside/actuators.hpp:107]: Yeah agree since we set register select each time we read anyways

**Review by BluCodeGH — APPROVED** [review]
Approving pending testing

**Pull request merged**